app/cdap/components/Replicator/Create/Content/Assessment/index.tsx

```diff
@@ -97,6 +102,7 @@ const AssessmentView: React.FC<ICreateContext & WithStyles<typeof styles>> = ({
   const [error, setError] = React.useState(null);
   const [schemaErrorCount, setSchemaErrorCount] = React.useState(0);
   const [view, setView] = React.useState(VIEWS.tables);
+  const [errorFeaturesCount, setErrorFeaturesCount] = React.useState(0);
```

Comment:
Import react, {useState} from 'react'; - i know it was previously done with React.useState but any time you alter the code please update the rest of the useState calls.

Also if you alter code with an <If></If> component, please remove it that replace it with { condition && <>}.

Last thing - if you have more than 4 useState calls, please replace it with a useReducer (don't do it on this component) for new components.

Reply:
Thanks! Updated